source/404.md

```diff
@@ -6,16 +6,14 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed.  
-Use menu above to navigate or go to [home page](/).  
-If you noticed any site problem, typos, etc on any page, 
-please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
+The page was moved or never existed. Use menu above to navigate or go to [home page](/).
+If you noticed any site problem (typos, etc on any page), please correct via [pull requesting](https://github.com/apache/struts-site/compare).
 
```

Comment:
https://github.com/apache/struts-site/compare is not a way to start PR, but editing is.